### ISSUE TITLE
Project Proposal Feedback

### ISSUE THREAD

**Issue by poojapathak1725** [COGS118A/Group038-Wi23#1]
~Project Proposal Grade - 7/9~
Updated Project Proposal Grade - 9/9

Title and Abstract (1/1) 
Please include an informative title for your project. Good - mention which model(s) you are planning to build. 

Background (1/1) - 
Good. Include a line at the end of this section that describes how your approach differs from the prior ones, and why you think your way would contribute to learning. 

Research Problem Statement (& Significance, Purpose of Project) (1/1.5)-
What kind of neural network do you plan to use for your project? Please specify. (-0.5)
Additionally, we expect to see a comparison and contrast of a couple of models (along with cross-validation, hyper param tuning etc.) per topic. 

Data (1/1.5)- 
 I would suggest rewriting this section in bullet points for better legibility. 
What kinds of feature engineering techniques do you plan to use? Any one-hot-encoding for the features? Scaling/Normalizing features? How will you deal with class imbalance issues - if any? Please specify.  (-0.5)

Proposed Solution (1/1.5) - 
OK - I would suggest trying some supervised learning approaches as well such as RF, SVM, etc. and comparing the performance of these approaches with the neural network and the benchmark. We need to see a compare/contrast of at least 3 models based on their performance on the evaluation metrics. 
Mention the train/test split, cross-validation techniques, hyperparam tuning etc. in this section as well. (-0.5)

Metrics (0.25/0.5) - 
OK. Mention which kinds of mistakes are more costly, and include mathematical formulae for the metrics. (-0.25)

Ethics and Privacy (-0.25) - 
Mention that all patient data is anonymized in your dataset so no privacy is violated. You mention a risk with False Negatives in your project - how will you deal with this? What are the potential harms of your project? Keep adding to this section. (-0.25)

Team Expectations (0.25/0.25) - 
 OK. 

Timeline  (0.25/0.25) - 
OK. 

Other Comments - Good start, I think you could try other supervised learning approaches to this topic as well. Look forward to seeing your project. 

You can reply to this feedback below. Make sure to implement all of the feedback mentioned here so you can make up these points during Checkpoint. Contact me anytime if you want help improving your project or have any questions at all!
